python/lsst/ctrl/pool/pool.py

```diff
@@ -300,6 +300,30 @@ def broadcast(self, value, root=0):
         with PickleHolder(value):
             return super(Comm, self).bcast(value, root=root)
 
+    def scatter(self, dataList, root=0, tag=0):
+        """Scatter data across the nodes
+
+        The default version apparently pickles the entire 'dataList',
+        which can cause errors if the pickle size grows over 2^31 bytes
+        due to fundamental problems with pickle in python 2. Instead,
+        we send the data to each slave node in turn; this reduces the
+        pickle size.
+
+        @param dataList  List of data to distribute; one per node
+            (including root)
+        @param root  Index of root node
+        @param tag  Message tag (integer)
+        @return  Data for this node
+        """
```

Comment:
I would, but I think using the old style is preferred to using heterogeneous styles in the same file, and also preferred to investing time in updating the entire file to use numpydoc.